TypedSyntax/src/node.jl

```diff
@@ -306,15 +307,22 @@ end
 
 function getsrc(@nospecialize(f), @nospecialize(t))
     srcrts = code_typed(f, t; debuginfo=:source, optimize=false)
-    return only(srcrts)
+    src, rt = only(srcrts)
+    if hasfield(typeof(src), :parent) && src.parent !== nothing
```

Comment:
```suggestion
    if src.parent !== nothing
```
`codeinfo.parent` didn't get deleted.